src/GradeScale.ts

```diff
@@ -31,6 +31,15 @@ export const GradeScales = {
 
 export type GradeScalesTypes = typeof GradeScales[keyof typeof GradeScales]
 
+export const ConversionGroups = {
+  AID: 'aid',
+  FREE: 'free',
+  BOULDERING: 'bouldering',
+  ICE: 'ice'
+} as const
+
+export type ConversionGroupsTypes = typeof ConversionGroups[keyof typeof ConversionGroups]
+
```

Comment:
@musoke is this what you meant for enum or did you refer to https://www.typescriptlang.org/docs/handbook/enums.html ?

I've opted for this to be stay consistent to the above type.  Let me know if this is okay

Reply:
I was referring to your link, but this might do the trick too